fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -648,7 +648,7 @@ public enum Counts implements Count {
         /** The number of times that an index entry does not point to a valid record. */
         BAD_INDEX_ENTRY("number of occurrences of bad index entries", false),
         /** The number of record keys repaired by {@link FDBRecordStore#repairRecordKeys(byte[], com.apple.foundationdb.record.ScanProperties)}. */
-        REPAIR_RECORD_KEY("repair record key", false),
+        REPAIR_RECORD_KEY("repair record key", false, null, true),
```

Comment:
This one is delayed because its only use is here: https://github.com/FoundationDB/fdb-record-layer/blob/09b9d9d05c2e392334f9f1c73bb1980cd87eca6a/fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java#L4537

And it only is incremented when a transaction decides to make some mutations (so, if the transaction doesn't commit, the keys aren't repaired). Note that if the transaction fails due to, say, a conflict and is retried, then only publishing after commit means that the total count of this metric at the end won't double count keys that are "repaired" during the failed commit.

There's an argument that something like `SAVE_RECORD` should also be marked as delayed, but I didn't because `SAVE_RECORD` is an `Event` with an associated time, and it can be useful to know how much time is spent doing a save record even if the transaction fails.

That being said, I'm not too attached to this one being delayed and could change it